tests/quantization/test_hawq_precision_init.py

```diff
@@ -566,8 +566,8 @@ def precision_init_dumping_worker(gpu, ngpus_per_node, config, tmp_path):
     model = safe_thread_call(partial(mobilenet_v2, pretrained=True))
     model.eval()
     criterion = torch.nn.MSELoss().cuda(config.gpu)
-    config = register_default_init_args(config, data_loader, criterion,
-                                        autoq_eval_fn=lambda *x: 0, autoq_eval_loader=data_loader)
+    config = register_default_init_args(config, data_loader, None, criterion,
+                                        autoq_eval_fn=lambda *x: 0, val_loader=data_loader)
```

Comment:
Rewrite this using keyword arguments instead of having to specify `None` as an argument.

Reply:
Done